requirements.txt

```diff
@@ -19,7 +18,6 @@ numpy==1.19.0
 persistent>=4.2.0
 plyvel
 py-flags
-py-evm==0.3.0a13
```

Comment:
native contracts depend on py-evm right?

Reply:
oh oops, didn't mean to include these changes